tests/unit/errors_test.py

```diff
@@ -117,3 +118,18 @@ def test_create_unexpected_kwargs_error_single(self):
     def test_create_unexpected_kwargs_error_multiple(self):
         e = create_unexpected_kwargs_error('f', {'foo': 'bar', 'baz': 'bosh'})
         assert str(e) == "f() got unexpected keyword arguments 'baz', 'foo'"
+
+
+class CreateErrorFromException(unittest.TestCase):
```

Comment:
Let's add this test to the `APIErrorTest` class instead. :)

Reply:
Done